docs/user/authentication_and_authorization.md

```diff
@@ -49,13 +49,13 @@ It is available under `/ngsi-ld/v1/entityAccessControl/entities` and can be call
 
 The following request parameters are supported: 
 
-* `attrs`: restrict returned entities to the ones with a specific right. Only `rCanRead`, `rCanWrite` and `rCanAdmin` are accepted. A list is accepted (e.g, `attrs=rCanRead,rCanWrite`). This request parameter has no effect when user has the _stellio-admin_ role
+* `attrs`: restrict returned entities to the ones with a specific right. Only `canRead`, `canWrite`, `canAdmin` and `isOwner` are accepted. A list is accepted (e.g, `attrs=canRead,canWrite`). This request parameter has no effect when user has the _stellio-admin_ role
 * `type`: restrict returned entities to given entity types
 * `id`: restrict returned entities to given ids (comma separated list of strings)
 
 There are several possible answers:
 
-* If user is not admin of the entity but it has a right on it (`rCanRead` or `rCanWrite`), the response body will be under this form:  
+* If user is not admin nor owner of the entity but it has a right on it (`canRead` or `canWrite`), the response body will be under this form:  
```

Comment:
since user is a person , I think saying _they have a right on it  _ will be better

Reply:
user can be an user or a client (with a service account). maybe user should instead be renamed to subject.